paket.lock

```diff
@@ -39,10 +43,14 @@ NUGET
     Fable.Promise (3.1)
       Fable.Core (>= 3.1.5) - restriction: >= netstandard2.0
       FSharp.Core (>= 4.7.2) - restriction: >= netstandard2.0
-    Fable.React (7.4)
+    Fable.React (7.4) - restriction: >= netstandard2.0
       Fable.Browser.Dom (>= 2.0.1) - restriction: >= netstandard2.0
       Fable.Core (>= 3.1.5) - restriction: >= netstandard2.0
       FSharp.Core (>= 4.7.2) - restriction: >= netstandard2.0
+    Fable.React.Types (18.0.0-beta-001)
```

Comment:
Why does this need to be `beta`?

Reply:
I always push a beta version first just in case 😅 But you're right, a stable version should be fine as we've been using the bindings for a while now. I will update it 👍